api/v1/storagecluster_types.go

```diff
@@ -67,6 +67,8 @@ type StorageClusterSpec struct {
 	// ArbiterSpec specifies the storage cluster options related to arbiter.
 	// If Arbiter is enabled, ArbiterLocation in the NodeTopologies must be specified.
 	Arbiter ArbiterSpec `json:"arbiter,omitempty"`
+	// DataMirroring specifies data mirroring configuration for the storage cluster.
+	DataMirroring MirroringSpec `json:"dataMirroring,omitempty"`
```

Comment:
I wonder if "data" in `DataMirroring` is not implied. What about just `Mirroring`?

Reply:
I started with `Mirroring`, then switched to `DataMirroring` because I felt the need to be explicit about it.
But, since `Mirroring` is used everywhere else, I guess it's better to be consistent.